guide/voice/optimisation-and-troubleshooting.md

```diff
@@ -2,9 +2,9 @@
 
 <branch version="11.x">
 
-The Discord.js voice system allows your bot to join voice channels and play audio. This guide will teach you how to make simple music bots, and tips and tricks to optimize performance!
+The Discord.js voice system allows your bot to join voice channels and play audio. This guide will teach you how to make simple music bots and tips and tricks to optimize performance!
```

Comment:
```suggestion
The Discord.js voice system allows your bot to join voice channels and play audio. This guide will teach you how to make simple music bots, as well as tips and tricks to optimize performance!
```

Reply:
"and" suffices. It's concise and gets the same point across where "as well as" contributes to wordiness.

Reply:
Updated my suggestion